djangocms_frontend/contrib/modal/forms.py

```diff
@@ -0,0 +1,167 @@
+from ...helpers import first_choice
+from ...models import FrontendUIItem
+from .. import modal
+from .constants import (
+    MODAL_CENTERED_CHOICES,
+    MODAL_FULLSCREEN_CHOICES,
+    MODAL_INNER_TYPE_CHOICES,
+    MODAL_SIZE_CHOICES,
+)
+
+# TODO leaving this comment for now
+# data-bs-toggle="modal" data-bs-target="#modalExample"
+# aria-expanded="false" aria-controls="modalExample">
+# data-bs-target can also be classes
+# data-bs-parent links to the wrapper modal
+# <div class="modal" id="modalExample">
+
+mixin_factory = settings.get_forms(modal)
+
+
+class ModalForm(mixin_factory("Modal"), EntangledModelForm):
+    """
+    Component > "Modal" Plugin
+    https://getbootstrap.com/docs/5.0/components/modal/
+    """
+
+    class Meta:
+        model = FrontendUIItem
+        entangled_fields = {
+            "config": [
+                "modal_siblings",
+                "attributes",
+            ]
+        }
+        untangled_fields = ("tag_type",)
+
+    modal_siblings = forms.CharField(
+        label=_("Siblings"),
+        initial=".card",
+        required=False,
+        help_text=_("Element to be used to create a modal."),
+    )
+    attributes = AttributesFormField()
+    tag_type = TagTypeFormField()
+
+
+class ModalTriggerForm(mixin_factory("ModalTrigger"), EntangledModelForm):
+    """
+    Component > "Modal Trigger" Plugin
+    https://getbootstrap.com/docs/5.0/components/modal/
+    """
+
+    class Meta:
+        model = FrontendUIItem
+        entangled_fields = {
+            "config": [
+                "trigger_identifier",
+                "attributes",
+            ]
+        }
+        untangled_fields = ("tag_type",)
+
+    trigger_identifier = forms.SlugField(
+        label=_("Unique identifier"),
+        required=True,
+        help_text=_("Identifier to connect trigger with container."),
+    )
+    attributes = AttributesFormField()
+    tag_type = TagTypeFormField()
+
+
+class ModalContainerForm(mixin_factory("ModalContainer"), EntangledModelForm):
+    """
+    Component > "Modal Container" Plugin
+    https://getbootstrap.com/docs/5.0/components/modal/
+    """
+
+    class Meta:
+        model = FrontendUIItem
+        entangled_fields = {
+            "config": [
+                "container_identifier",
+                "attributes",
+                "modal_centered",
+                "modal_static",
+                "modal_scrollable",
+                "modal_size",
+                "modal_fullscreen",
+            ]
+        }
+        untangled_fields = ("tag_type",)
+
+    container_identifier = forms.SlugField(
+        label=_("Unique identifier"),
+        required=True,
+        help_text=_("Identifier to connect trigger with container."),
+    )
+
+    modal_centered = forms.ChoiceField(
+        label=_("Centered"),
+        choices=settings.EMPTY_CHOICE + MODAL_CENTERED_CHOICES,
+        required=False,
+    )
+
+    modal_static = forms.BooleanField(
+        label=_("Static"),
```

Comment:
```suggestion
        label=_("Static backdrop"),
```
I assume this refers to https://getbootstrap.com/docs/5.2/components/modal/#static-backdrop